tests/features/bpmsuite/businesscentral/configuration.feature

```diff
@@ -0,0 +1,11 @@
+@jboss-bpmsuite-7/bpmsuite70-businesscentral-openshift @jboss-bpmsuite-7/bpmsuite70-businesscentral-monitoring-openshift
+Feature: Business Central configuration tests
+
+  # https://issues.jboss.org/browse/CLOUD-2221
+  Scenario: Check KieLoginModule is configured
+  Given XML namespaces
+        | prefix | url                           |
```

Comment:
It is not necessary as you aren't using xml namespace on the tests.

Reply:
Ah, good point, thanks. I was trying to use the xpath way, but that's actually not necessary for this simple case. Will remove that.